my_list.rb

```diff
@@ -0,0 +1,17 @@
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
```

Comment:
- [ ] @PedroH-Neves and @najibullahjafari Great work adding MyList class, however, you forgot to import the MyEnumerable module eg ` require_relative 'my_enumerable'', and this is making your example not run.
`